cmd/config-sync/config_sync_test.go

```diff
@@ -0,0 +1,347 @@
+package main
+
+import (
+	"github.com/skupperproject/skupper/client"
+	"github.com/skupperproject/skupper/pkg/event"
+	"github.com/skupperproject/skupper/pkg/qdr"
+	"github.com/skupperproject/skupper/pkg/utils"
+	"gotest.tools/assert"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+	k8stesting "k8s.io/client-go/testing"
+	"os"
+	"testing"
+)
+
+const TEST_TLS_DIRECTORY = "~/.test/skupper-router/tls"
```

Comment:
I would not write to `~`. It might be better to use `./tmp` or some internal location inside the cloned repo.